crates/forge/benches/my_benchmark.rs

```diff
@@ -0,0 +1,213 @@
+            let balance = dispatcher.get_balance();
+            dispatcher.decrease_balance(100);
+            let balance = dispatcher.get_balance();
+
+            start_prank(contract_address, 1234.try_into().unwrap());
+            start_roll(contract_address, 234);
+            start_warp(contract_address, 123);
+
+            let (x, y, z) = dispatcher.interact_with_state();
+        }
+        "#
+        ),
+        Contract::new(
+            "HelloStarknet",
+            indoc!(
+                r#"
+                #[starknet::contract]
+                mod HelloStarknet {
+                    use box::BoxTrait;
+                    use starknet::ContractAddressIntoFelt252;
+                    use starknet::ContractAddress;
+                    use integer::U64IntoFelt252;
+                    use option::Option;
+                    use traits::Into;
+
+                    #[storage]
+                    struct Storage {
+                        balance: felt252,
+                    }
+        
+                    // Increases the balance by the given amount.
+                    #[external(v0)]
+                    fn increase_balance(ref self: ContractState, amount: felt252) {
+                        self.balance.write(self.balance.read() + amount);
+                    }
+        
+                    // Decreases the balance by the given amount.
+                    #[external(v0)]
+                    fn decrease_balance(ref self: ContractState, amount: felt252) {
+                        self.balance.write(self.balance.read() - amount);
+                    }
+
+                    //Get the balance.
+                    #[external(v0)]
+                    fn get_balance(self: @ContractState) -> felt252 {
+                        self.balance.read()
+                    }
+
+                    #[external(v0)]
+                    fn interact_with_state(self: @ContractState) -> (felt252, felt252, felt252) {
+                        let caller_address: felt252 = starknet::get_caller_address().into();
+                        let block_number = starknet::get_block_info().unbox().block_number;
+                        let block_timestamp = starknet::get_block_info().unbox().block_timestamp;
+
+                        (caller_address, block_number.into(), block_timestamp.into())
+                    }
+                }
+                "#
+            )
+        )
+    )
+}
+
+fn declare_and_interact(test: &TestCase) {
+    let result = run_test_case(test);
+
+    assert_passed!(result);
+}
+
+fn criterion_benchmark(c: &mut Criterion) {
+    let declare_and_interact_input = setup_declare_and_interact();
+    let collect_tests_input = setup_collect_tests();
+    let compile_tests_input = setup_compile_tests();
+
+    let mut group = c.benchmark_group("benchmark-normal-flow");
+    group.sampling_mode(SamplingMode::Flat); // https://bheisler.github.io/criterion.rs/book/user_guide/advanced_configuration.html#sampling-mode
```

Comment:
```suggestion
    group.sampling_mode(SamplingMode::Flat); // (Relatively) Long running benchmark
```
Plz don't make people open up browser to find out what the code does